src/Server/RestRouterHandlers/ColumnDefinition.cpp

```diff
@@ -65,6 +69,10 @@ String getUpdateColumnDefination(const Poco::JSON::Object::Ptr & payload, String
     {
         update_segments.push_back(" MODIFY COLUMN " + column_name + " DEFAULT " + payload->get("default").toString());
     }
+    else if (payload->has("alias"))
+    {
+        update_segments.push_back(" MODIFY COLUMN " + column_name + " ALIAS " + payload->get("alias").toString());
+    }
```

Comment:
MODIFY COLUMN IF EXISTS ?

Reply:
Let‘s add this when solving the conflicts later.